src/appliance-onboarding-script/appliance_setup/avs/avsarconboarder/constants/Constant.py

```diff
@@ -30,4 +30,4 @@
 CLUSTER_SIZE = "cluster_size"
 VNET_IP_CIDR = "VNET_IP_CIDR"
 DNS_SERVICE_IP = "DNS_Service_IP"
-VALID_LOCATIONS = "westeurope,eastus"
+VALID_LOCATIONS = "westeurope,eastus,uksouth"
```

Comment:
Will this location also be used to create the other Arm resources(Appliance etc.)?
Also, what is the experience for SEA and canadaCentral? Are they not listed here in valid_regions since we have enabled them only for internal customers?

Reply:
If they specify the location in the config, that will be used for arc resources otherwise the private cloud's location will be used. The same would be validated against this list.

Reply:
UKSouth too is not supported by arc, so would have a similar experience for that. Abandoning this PR.